### PR TITLE
Intermittent producer in multitrack

### PR DESCRIPTION
**Pull request by disa6302**
*Issue #, if available:*
#144 
*Description of changes:*
- Add multi track option for producer canary
- Include a job to run intermittent scenario in single track and multi track scenario
- Update commit hash to include latest fix

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: canary/producer-c/canary/KvsProducerSampleCloudwatch.cpp [hunk 1]
```diff
@@ -366,7 +378,16 @@ INT32 main(INT32 argc, CHAR* argv[])
...
             if (GETTIME() < runTill) {
+                frame.trackId = DEFAULT_VIDEO_TRACK_ID;
                 CHK_STATUS(putKinesisVideoFrame(streamHandle, &frame));
+
+                // Send frame on another track only if we want to run multi track. For the sake of
+                // multitrack, we use the same frame for video and audio and just modify the flags.
+                if (STRCMP(config.canaryTrackType, CANARY_MULTI_TRACK_TYPE) == 0) {
+                    frame.flags = FRAME_FLAG_NONE;
+                    frame.trackId = DEFAULT_AUDIO_TRACK_ID;
+                    CHK_STATUS(putKinesisVideoFrame(streamHandle, &frame));
```
Comment (MushMal) [inline canary/producer-c/canary/KvsProducerSampleCloudwatch.cpp:389]: Question. What happens if we get an error on putframe (or other API calls for that matter here). Do we end up just logging it or we have a way to output error metrics? I presume the latter but wanted to confirm.

**Review by MushMal — APPROVED** [review]
Nice!

File: canary/producer-c/canary/KvsProducerSampleCloudwatch.cpp (hunk 1 above)
Comment (disa6302) [reply canary/producer-c/canary/KvsProducerSampleCloudwatch.cpp:389]: You are right. PutFrame error metrics are emitted in the canary to cloudwatch.

**Review by hassanctech — APPROVED** [review]
Great!

**Pull request merged**